dspace/config/modules/bulkedit.cfg

```diff
@@ -20,8 +20,10 @@
 
 # Metadata elements to exclude when exporting via the user interfaces, or when using the
 # command line version and not using the -a (all) option.
-# bulkedit.ignore-on-export = dc.date.accessioned, dc.date.available, \
-#                    dc.date.updated, dc.description.provenance
+#bulkedit.ignore-on-export = dc.date.accessioned
+#bulkedit.ignore-on-export = dc.date.available
+#bulkedit.ignore-on-export = dc.date.updated
+#bulkedit.ignore-on-export = dc.description.provenance
```

Comment:
Is this change strictly necessary? Looking at other usage of getArrayProperty(), the standard "comma separated list" seems to be the usual approach.

See existing usage, and the unit tests:
`l.put("sample.array", "itemA,itemB,itemC");`
...
`// Pass in default value
        array = configurationService.getArrayProperty("sample.array", new String[]{"Hey"});
        // Assert default value not used, since property exists
        assertEquals(3, array.length);`

This is not a really big deal but as far as I can tell, this is an unnecessary change that deviates from the typical way multiple fields are configured in DSpace.

Reply:
No, multiple assignments are the proper way to express multiple values to Commons Configuration in a Properties configuration file.  The unwieldy old comma-list style only continues to work so long as the PropertiesConfiguration uses DefaultListDelimiterHandler.  I'm actually surprised that this works -- Commons Configuration API docs say that by default list splitting is disabled.  (Maybe that changed between CC 2.3 and 2.6.)

We should be taking these opportunities to normalize the expression of such lists, and removing our own list-parsing code where we find it.

Reply:
Apologies. I was just comparing it to the rest of DSpace configuration. I'll resolve this now, thanks